src/app/dashboard/Missions/DetailsView.jsx

```diff
@@ -224,13 +230,19 @@ const MissionDetailsCard = ({ mission, toEditView, toListView }) => {
             <Label classes={classes}>Notes</Label>
             <Row classes={classes}>{mission?.deliveryNotes || "No additional informations"}</Row>
             <Box className={classes.buttonGroupBox}>
-              <Box className={classes.buttonBox} onClick={toEditView}>
-                <EditIcon className={classes.icon} /> Edit
-              </Box>
-              <Box className={classes.buttonBox} onClick={() => console.log("cancel")}>
-                <CancelIcon className={classes.icon} />
-                Cancel Mission
-              </Box>
+              <Grid container direction="row" spacing={2} alignItems="center">
+                <Grid item>
+                  <Box item className={classes.buttonBox} onClick={toEditView}>
+                    <EditIcon className={classes.icon} /> Edit
+                  </Box>
+                </Grid>
+                <Grid item>
+                  <Box item className={classes.buttonBox} onClick={() => console.log("cancel")}>
```

Comment:
Clicking on this box console logs?

Reply:
I'm actually not clear on what should happen when a mission is Cancelled.  Goign to ask on Slack.